projects/map-ol/src/lib/helpers.ts

```diff
@@ -0,0 +1,17 @@
+export function getUrlFromUri(uri: string) {
```

Comment:
This function I would place in `@dlr-eoc/utils-browser` then add it as a dependency to `@dlr-eoc/map-ol`

Reply:
Removed since no longer required after [this comment](https://github.com/dlr-eoc/ukis-frontend-libraries/pull/76#discussion_r601206453)